RELEASE/scripts/autoscend/quests/level_any.ash

```diff
@@ -1104,3 +1104,52 @@ boolean LX_dronesOut()
 	}
 	return false;
 }
+
+boolean LX_lastChance()
+{
+	//miscellaneous calls that aren't powerlevelling but need to be done at some point based on certain conditions
+	if(get_property("screechDelay").to_boolean())
+	{
+		auto_log_warning("Patriotic Eagle's screech banished something we need and we can't adventure anywhere else");
+		while(get_property("screechCombats").to_int() > 0 && my_adventures() > 2)
+		{
+			handleFamiliar($familiar[Patriotic Eagle]); //force eagle to be used
+			if(LX_getDigitalKey() || LX_getStarKey())
+			{
+				continue;
+			}
+			else
+			{
+				if(can_adventure($location[Cobb\'s Knob Harem]))

```

Comment:
this can do any quest that doesn't rely on the phylum currently banished (although presumably this is caught by previous L tasks).

it could also powerlevel or unlock the island if not already unlocked or do crafting (or even rest if we have cincho and need NCs).

given it is only in the while loop while screechCombats is >0 (i.e. while we can't screech), going to screech zones is probably unhelpful.